src/server/ds/main.c

```diff
@@ -17,6 +17,7 @@ struct data_processor worker_processor = {
     ds_process_read,
     ds_process_write,
     ds_process_error,
+    true
```

Comment:
.running = true

That was my original suggestion. Your choice though.

Reply:
Done.